.github/workflows/ci.yml

```diff
@@ -69,6 +70,13 @@ jobs:
             - { prettyname: Linux, fullname: ubuntu-latest }
           threadingMode: ['SingleThread', 'MultiThreaded']
     timeout-minutes: 60
+
+    services:
+      httpbin:
+        image: kennethreitz/httpbin
+        ports:
+          - 80:80
```

Comment:
forgot :443? seems to be the only issue

Reply:
I closed it mostly because it doesn't work on windows/macOS runners. Thoughts on that?

Reply:
Dunno. I'm fine with it only running in one place if that helps.

Reply:
It also gets a little complicated actually doing that, because I can't find any way to do conditional `services`, so it looks like I'd have to split out the build matrix. Just gonna keep this one on the backburner for now, it was a <5 minute test to see if it works and the answer is "maybe".